src/griffe/dataclasses.py

```diff
@@ -428,6 +432,22 @@ def is_kind(self, kind: str | Kind | set[str | Kind]) -> bool:
             kind = Kind(kind)
         return self.kind is kind
 
+    @cached_property
+    def inherited_members(self) -> dict[str, Alias]:
+        """Members that are inherited from base classes."""
+        if not isinstance(self, Class):
+            return {}
+        inherited_members = {}
+        for base in reversed(self.mro()):
+            for name, member in base.members.items():
+                inherited_members[name] = Alias(name, member, parent=self)  # type: ignore[arg-type]
+        return inherited_members
+
+    @property
+    def all_members(self) -> dict[str, Object | Alias]:
```

Comment:
I considered changing `members` to a property returning all members, and adding a `declared_members` property, but that would be a breaking change.